### DISCUSSION TITLE
What causes slow avatar speed?

### DISCUSSION THREAD

**Discussion by hilarytsai** [mozilla/hubs#2924] [General Questions & Discussion] [answered]
I've been told there is no real way to adjust the avatar's speed except for the user preferences, but even at max speed, my movement is very slow in this scene.  Is this caused by a large scene?   Does changing the floor plan or making my floor smaller make any difference? It doesn't seem to; but I need to fix this somehow because the speed really is very slow.

**Comment by misslivirose** [comment, marked as answer]
I see this when a scene is scaled very large - it's not actually changing the avatar speed, but because the distances are larger than expected, they appear to move super slowly. It might also be helpful to confirm that the avatar spawn points aren't scaled down. If you have a link to the scene we can give more specific recommendations.

**Reply by hilarytsai** [reply]
@misslivirose  It is indeed a large scene. But to scale everything down, I'd have to adjust every individual item proportionately, right? Or is there a way to scale everything down at the same time? Here is my scene:( https://hubs.mozilla.com/CEyUmoD/bossy-accurate-camp) Thank you for taking a look!

**Reply by hilarytsai** [reply]
oh, I realized you probably meant like a scene you can edit and see settings for. Here it is:  https://hubs.mozilla.com/scenes/Npsa6z6

**Reply by cvanmeurs** [reply]
Neat looking scene!  But it does appear that everything is out of scale and way too large.  For example when I walk up to the park bench it's towering over my head.  Even with your spawn points set at 200% scale, everything is still too large (and that's not the best way to fix it in the first place).

There are a couple reasons this could be happening:

1)  Your items are not modeled to scale.  Check to be sure that your park bench looks the appropriate size when placed next to a 6-foot tall (or ~2-meter tall) proxy person model.  Or if you know how to measure an item in your 3D package, measure it to make sure the assets are the proper size.

2)  If your items are the proper size in your 3D software, it could be that your export settings are not exporting your models at the proper size.  I'd recommend making something like a default 1-foot by 1-foot cube (or whatever unit of measure you are used to) and bringing that into an empty scene and verifying that it looks like the correct scale.  If it's way too large, then something is wrong with your export settings.

If I remember correctly, you're using Cinema 4D?
